### PR TITLE
Used event delegation for tooltips

### PR DESCRIPTION
**Pull request by m-voloshin**
Tooltips work better, but there is an [issue in Chrome](http://take.ms/rLV52).

### CODE REVIEW

**Comment by lyubomir** [comment]
Hey, @m-voloshin! I'll need more details from you before I merge your PR:

- The sentence that is the subject/title of this PR/commit "Used event delegation for tooltips" doesn't mean much to me, I'm afraid. While I Googled and skimmed through "jquery event delegation", I don't find this subject/title useful. It states a means, not a goal so it leaves me clueless as to why you're proposing these modifications to be merged. Please provide a subject which states the goal of your PR/commit. For example, 'Fix tooltips that...', 'Hide tooltips...', etc. Thea means belong in the source code and a short summary of the means may appear in the PR/commit description.

- The sentence that is the description of this PR "Tooltips work better, but there is an issue in Chrome" doesn't mean much to me, I'm afraid. If tooltips didn't work better, I wouldn't even consider merging this PR so please drop the "Tooltips work better." As to "there is an issue in Chrome," I watched the capture you provided at least five times and didn't understand what the "issue in Chrome" is so could you please describe it with sentences (regardless of the fact that a picture/video is worth a thousand words.)

File: modules/UI/util/UIUtil.js [hunk 1]
```diff
@@ -103,13 +103,17 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
...
+        });
         APP.translation.translateElement($(element));
 
         AJS.$(element).tooltip({
             gravity: positions[position],
             title: this._getTooltipText.bind(this, element),
-            html: true
+            html: true,
+            live: true,
+            hoverable: false
```
Comment (lyubomir) [inline modules/UI/util/UIUtil.js]: Isn't the default value of `hoverable` `false` already? If it is, could you please not specify it. Or is it really necessary?

File: modules/UI/util/UIUtil.js [hunk 2]
```diff
@@ -103,13 +103,17 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
...
+            this.setAttribute("data-i18n", "[content]" + key);
+        });
         APP.translation.translateElement($(element));
 
         AJS.$(element).tooltip({
             gravity: positions[position],
             title: this._getTooltipText.bind(this, element),
-            html: true
+            html: true,
+            live: true,
```
Comment (lyubomir) [inline modules/UI/util/UIUtil.js]: I couldn't find any documentation for `live` and had to look at the implementation to get a (vague) idea. Could you please provide a source code comment on `live` explaining why it's necessary, the problem that it fixes, etc.?

File: modules/UI/videolayout/RemoteVideo.js [hunk 3]
```diff
@@ -230,6 +230,7 @@ RemoteVideo.prototype.remove = function () {
     if (this.container.parentNode) {
         this.container.parentNode.removeChild(this.container);
     }
+    UIUtils.validateTooltips();
```
Comment (lyubomir) [inline modules/UI/videolayout/RemoteVideo.js]: Why do you call `UIUtils.validateTooltips` here? Why do you call it here only? I believe that's something that deserves a source code comment.

File: modules/UI/util/UIUtil.js (hunk 1 above)
Comment (m-voloshin) [inline modules/UI/util/UIUtil.js]: Unfortunately AUI team prefers hoverable tooltips [http://take.ms/z9eG3](http://take.ms/z9eG3)

File: modules/UI/util/UIUtil.js (hunk 2 above)
Comment (m-voloshin) [inline modules/UI/util/UIUtil.js]: Please, check my new PR: [https://github.com/jitsi/jitsi-meet/pull/939](https://github.com/jitsi/jitsi-meet/pull/939)

File: modules/UI/videolayout/RemoteVideo.js (hunk 3 above)
Comment (m-voloshin) [inline modules/UI/videolayout/RemoteVideo.js]: You are right, provided implementation is incomplete, please check [updated version](https://github.com/jitsi/jitsi-meet/pull/939).

**Comment by m-voloshin** [comment]
Provided updated implementation: [https://github.com/jitsi/jitsi-meet/pull/939](https://github.com/jitsi/jitsi-meet/pull/939)

**Pull request closed without merging**